src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/types.ts

```diff
@@ -31,20 +31,13 @@ export type TransportIconModeType = Omit<
   transportSubMode?: Required<TransportModes>['transportSubModes'][0];
 };
 
-export type TransportModeFilterOptionType = {
-  id: string;
-  icon: TransportIconModeType;
-  text: LanguageAndTextType[];
-  description?: LanguageAndTextType[];
-  modes: TransportModes[];
-};
-
-export type TravelSearchFiltersType = {
-  transportModes?: TransportModeFilterOptionType[];
-};
+export {
+  TravelSearchFiltersType,
+  TravelSearchTransport,
+} from '@atb-as/config-specs';
 
 export type TransportModeFilterOptionWithSelectionType =
-  TransportModeFilterOptionType & {selected: boolean};
+  TravelSearchTransport & {selected: boolean};
```

Comment:
May look at the naming here. Now it is a maybe illogical "naming-jump" from `TravelSearchTransport` to `TransportModeFilterOption`.

Reply:
I was trying to name based on config settings instead of what it was named as in the app, but could also inverse that 🤔 But won't be able to rename in firestore though.

Reply:
I think just using `TravelSearchTransportWithSelectedType` is good. The main thing is that it is the same type name, just with the selected field.

Reply:
Ahh, I see you have made the change the other way around. Just as good :+1:

Reply:
It should be named the same as before now FYI